draft/draft-ietf-rtgwg-qos-model.xml

```diff
@@ -222,45 +222,41 @@
     </section>
     
     <section title="QoS Model Design">
-      <t>A classifier consists of packets which may be grouped when a
-      logical set of rules are applied on different packet header
-      fields. The grouping may be based on different values or range
-      of values of same packet header field, presence or absence of
-      some values or range of values of a packet field or a
-      combination thereof. The QoS classifier is defined in the
+      <t> A classifier consists of set of filters and an operation
+      that is performed on those filters. Each filter defines a rule
+      on a particular packet header or packet metadata field that
+      dictate how the packet will be classified based on that
+      particular field, e.g. whether it will be classified based on
+      DSCP, or will be classified based on source IPv4 address
+      prefix. The operation defines whether these filters are
+      logically AND or OR. The filters may be based on a combination
+      of values or range of values of different packet header fields
+      or packet metadata fields. The QoS classifier is defined in the
       ietf-traffic-policy module.</t>
 
-      <t>A classifier entry contains one or more packet conditioning
-      functions. A packet conditioning function is typically based on
-      direction of traffic and may drop, mark or delay network
-      packets. A set of classifier entries with corresponding
-      conditioning functions when arranged in order of priority
-      represents a QoS policy. A QoS policy may contain one or more
-      classifier entries. These are defined in ietf-traffic-policy
-      module.</t>
-
-      <t>Actions are configured in line with respect to the policy
-      module.  These include marking, dropping or shaping. Actions are
-      defined in the ietf-qos-action module.</t>
+      <t>One or more packet conditioning functions may be applied
+      to a classifier which may drop, mark, or delay packets
+      These are defined in the ietf-qos-action module. A set of
+      classifiers with corresponding conditioning functions when 
+      arranged in order of priority represent a QoS policy.
+      These policies are defined in ietf-traffic-policy module.</t>
 
-      <t>A meter qualifies if the traffic arrival rate is based on
-      agreed upon rate and variability. A meter is modeled based on
+      <t>The Meter meters each packet and passes the packet and the
```

Comment:
The use of capital letter Meter and Marker is inconsistent with the rest of the document. We would have update classifiers to Classifiers etc. Also we are not referring to a particular kind of meter. I would revert back to using meter and marker.

Also, can we update the definition of meter to say what you were saying before, which is "The meter qualifies if the traffic arrival rate is based on an agreed upon rate, and variability, and passes the metering result to the marker."

Reply:
I think there was a yang doctor comment to use the definition from existing RFC rather than defining our own..